bigtable/admin/table_admin.h

```diff
@@ -129,6 +130,103 @@ class TableAdmin {
  private:
   std::string CreateInstanceName() const;
 
+  /// A shortcut for the grpc stub this class wraps.
+  using StubType =
+      ::google::bigtable::admin::v2::BigtableTableAdmin::StubInterface;
+
+  /**
+   * Determine if @p T is a pointer to member function with the expected
+   * signature for CallWithRetry().
+   *
+   * This is the generic case, where the type does not match the expected
+   * signature.  The class derives from std::false_type, so
+   * CheckSignature<T>::value is `false`.
+   *
+   * @tparam T the type to check against the
+   */
+  template <typename T>
+  struct CheckSignature : std::false_type {
+    /// Must define ResponseType because it is used in std::enable_if<>.
+    using ResponseType = int;
+  };
+
+  /// Discover if a pointer to member function has the expected signature.
```

Comment:
Do we need both a `///` and a `/** ... */` comment here? Is this not the same thing as the first line of the comment block below?

Reply:
Nope, that was a mistake.  Fixed.